spec/factories/jwt_api_entreprises.rb

```diff
@@ -33,10 +33,6 @@
       end
     end
 
-    trait :without_authorization_request_id do
-      authorization_request_id { nil }
-    end
-
```

Comment:
C'est pas directement lié à ça, mais est-ce qu'il est pas possible de clean le `after(:build)` plus haut pour cette factory, parce qu'il est bien brainfuck ?

Reply:
Oui c'est brainfuck je te l'accorde, c'était nécessaire pour la migration vers les webhooks Datapass. J'ai noté ça là https://github.com/etalab/admin_api_entreprise/issues/256 et du coup je passe en resolve, ça fera parti d'une autre PR.